jerry-core/jerry-api.h

```diff
@@ -68,6 +68,18 @@ typedef enum
   JERRY_ERROR_URI        /**< URIError */
 } jerry_error_t;
 
+typedef enum
+{
+  JERRY_FEATURE_ERROR_MESSAGES, /**< Error messages. */
```

Comment:
The comment style for fields does not start with capital letters and no full stop at the end. Please check the coding style guidelines.